Author: Niko Tyni <ntyni@debian.org>
Description: Properly call PERL_SYS_INIT3() to fix lockups on hppa with perl 5.10.0. (Closes: #486070)

This was always buggy but didn't bite until now. From 'perldoc perlembed':

The macros PERL_SYS_INIT3() and PERL_SYS_TERM() provide system-specific
tune up of the C runtime environment necessary to run Perl interpreters
--- a/src/speedy_backend_main.c
+++ b/src/speedy_backend_main.c
@@ -170,6 +170,8 @@
     int i;
     SigList sl;
 
+    PERL_SYS_INIT3(&argc, &argv, &_junk);
+
     speedy_util_unlimit_core();
 
     if (!(my_perl = perl_alloc()))
--- a/src/speedy_perl.c
+++ b/src/speedy_perl.c
@@ -405,6 +405,7 @@
 
 	perl_destruct(my_perl);
     }
+    PERL_SYS_TERM();
     speedy_util_exit(0,0);
 }
 
